sirepo/template/raydata.py

```diff
@@ -63,38 +62,29 @@
     ),
 )
 
-
-def background_percent_complete(report, run_dir, is_running):
-    def _png_filenames():
-        return [
-            pkio.py_path(f).basename for f in sorted(
-                glob.glob(str(run_dir.join('*.png'))),
-                key=os.path.getmtime
+def analysis_job_output_files(data):
+    def _filename_and_image(path):
+        return PKDict(
+            filename=path.basename,
+            image=pkcompat.from_bytes(
+                base64.b64encode(
+                    pkio.read_binary(path),
+                ),
             )
-        ]
+        )
 
-    def _sanitized_name(filename):
-        return sirepo.util.sanitize_string(filename) + 'Animation'
+    def _paths():
+        d = _dir_for_scan_uuid(_parse_scan_uuid(data))
+        for f in sorted(glob.glob(str(d.join('*.png'))), key=os.path.getmtime):
```

Comment:
add key to `pkio.sorted_glob` and this becomes:

```py
for f pkio.sorted_glob(d.join('*.png'), key='mtime'):
    yield f
```

I'm not sure `mtime` as a string is the best way to handle this, but `d.mtime` would not work. What you want is the method, or `key=lambda x: x.mtime()`, which is awkward. I think the string is ok, just have to call the method.

Reply:
7dc92d008